resources/authgear/templates/en/web/promote.html

```diff
@@ -45,32 +45,20 @@ <h1 class="margin-t-0 margin-b-20 primary-txt text-center text-xl font-bold">
 				{{ if eq .type "login_id" }}{{ if eq .login_id_key $.x_login_id_key }}
 				<input type="hidden" name="x_login_id_type" value="{{ .login_id_type }}">
 				{{ if eq .login_id_type "phone" }}
-					<div class="margin-b-20 phone-input">
-						<select class="select primary-txt" name="x_calling_code">
-							{{ range $.CountryCallingCodes }}
-							<option
-								value="{{ . }}"
-								{{ if $.x_calling_code }}{{ if eq $.x_calling_code . }}
-								selected
-								{{ end }}{{ end }}
-								>
-								+{{ . }}
-							</option>
-							{{ end }}
-						</select>
-						<input
-							tabindex="0"
-							autofocus
-							class="input text-input primary-txt"
-							type="text"
-							inputmode="numeric"
-							pattern="[0-9]*"
-							autocomplete="tel-national"
-							autocapitalize="none"
-							name="x_national_number"
-							placeholder="{{ template "phone-number-placeholder" }}"
-						>
-					</div>
+					<input
+						data-intl-tel-input="true"
+						data-intl-tel-input-class="margin-b-20 flex flex-col"
+						class="margin-b-20 primary-txt input text-input"
+						tabindex="0"
+						autofocus
+						type="text"
+						inputmode="numeric"
+						pattern="[0-9]*"
+						autocomplete="tel-national"
+						autocapitalize="none"
+						name="x_e164"
```

Comment:
I think this one should be `x_login_id` too

Reply:
Fixed. Sorry :(